### PR TITLE
update(validate and publish): UI alignment fix

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

UI style updates to validate and publish to fix alignment

## Why

To enhance the look and feel of the application

## Issue

ref: CPLP-2806

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx [hunk 1]
```diff
@@ -494,10 +497,15 @@ export default function CommonValidateAndPublish({
               (item === DocumentTypeId.ADDITIONAL_DETAILS ||
                 item === DocumentTypeId.APP_CONTRACT ||
                 item === DocumentTypeId.APP_TECHNICAL_INFORMATION) && (
-                <li key={item} className="document-list">
+                <li
+                  key={item}
+                  className="document-list"
+                  style={{ padding: '0 18px' }}
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx:503]: can you move this inline style to css file?

File: cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx [hunk 2]
```diff
@@ -494,10 +497,15 @@ export default function CommonValidateAndPublish({
...
+                >
                   <ArticleOutlinedIcon sx={{ color: '#9c9c9c' }} />
                   <button
                     className="document-button-link"
+                    style={{ fontSize: '16px' }}
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx:508]: can you move this inline style to css file?

File: cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx [hunk 3]
```diff
@@ -447,27 +447,30 @@ export default function CommonValidateAndPublish({
...
+              statusData.documents[
+                DocumentTypeText.CONFORMITY_APPROVAL_BUSINESS_APPS
+              ] &&
+              statusData.documents[
+                DocumentTypeText.CONFORMITY_APPROVAL_BUSINESS_APPS
+              ].map((item: DocumentData) => (
+                <li
+                  key={item.documentId}
+                  className="document-list"
+                  style={{ padding: '0 18px' }}
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx:460]: can you move this inline style to css file?

File: cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx [hunk 4]
```diff
@@ -447,27 +447,30 @@ export default function CommonValidateAndPublish({
...
+                >
+                  <ArticleOutlinedIcon sx={{ color: '#9c9c9c' }} />
+                  <button
+                    className="document-button-link"
+                    style={{ fontSize: '16px' }}
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx:465]: can you move this inline style to css file?

File: cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx [hunk 5]
```diff
@@ -415,7 +415,7 @@ export default function CommonValidateAndPublish({
               </div>
               {statusData?.privacyPolicies &&
               statusData?.privacyPolicies.length ? (
-                <div className="policies-list" style={{ maxWidth: '600px' }}>
+                <div className="policies-list" style={{ padding: '0 20px' }}>
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonValidateAndPublish.tsx:418]: can you move this inline style to css file?

**Review by manojava-gk — COMMENTED** [review]
small findings

**Review by oyo — APPROVED** [review]
lgtm

**Pull request merged**